hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/state/DispatchingEvmFrameState.java

```diff
@@ -79,8 +79,10 @@
  * TODO - get a little further to clarify DI strategy, then bring back a code cache.
  */
 public class DispatchingEvmFrameState implements EvmFrameState {
-    private static final Key HOLLOW_ACCOUNT_KEY =
-            Key.newBuilder().keyList(KeyList.DEFAULT).build();
+    // This key can no longer be an empty key list as it fails validation.  Use this key for now
```

Comment:
what does "for now" mean in this comment (i.e., is this a TODO or something else)?

Reply:
Yes.  This is something I need to discuss with Michael T when he gets back